security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,17 @@
+[id="security-container-signature"]
+= Container image signatures
+
+Red Hat delivers signatures for the images in the Red Hat Container Registries. Those signatures can be automatically verified when being pulled to {product-title} 4 clusters using the Machine Config Operator (MCO).
+
+Most of the images that comprise {product-title} are shipped from link:https://quay.io/[Quay.io], and only the release image is signed. Release images refer to the approved {product-title} images, offering a degree of protection against supply chain attacks. However, some extensions to {product-title} such as logging, monitoring, and service mesh are shipped as Operators from the Operator Lifecycle Manager (OLM). Those images ship from the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog Container images] registry.
```

Comment:
Not sure about _shipped_. Makes me think they are coming via FedEx. ;-)
I am not expert with images, but I see _[serves](https://docs.openshift.com/container-platform/4.7/architecture/architecture.html#architecture-key-features_architecture)_ used frequently with images from quay. 
_are served by_ ?
_Quay.io serves most of the images that comprise..._?